sdks/wasm/packager.cs

```diff
@@ -655,6 +655,8 @@ class WasmOptions {
 		}
 		if (ee_mode == ExecMode.AotInterp)
 			aot_args = "interp,";
+		if (ee_mode == ExecMode.Interp)
+			enable_zlib = true;
```

Comment:
was this related?

Reply:
This is needed because the runtime is compiled against zlib, so in reality this option always needs to be enabled. In AOT mode, the native linker can get rid of the references to zlib, but not in interpreter mode.